packages/terra-collapsible-menu-view/src/CollapsibleMenuView.module.scss

```diff
@@ -36,6 +36,10 @@
       display: none;
       width: 0;
     }
+
+    :focus {
+      outline-offset: var(--terra-collapsible-menu-view-menu-button-focus-outline-offset, -2px);
```

Comment:
Can you help me to understand how dashed outline is showing up for menu-button when it has not got outline style defined

Reply:
A menu button is also a face up item, so it will get it from there:
https://github.com/cerner/terra-framework/pull/1921/files/09658a0ea08182297ecc2d9446f77f2c9b2f46d1#diff-670832b407af1059b9923c758918150ee8ff8e90e069d00164ed6cd9ab68c9ffR64